src/components/Callout/CalloutMenu.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import MoreIcon from 'react-icons/lib/md/more-vert'
+import Callout from './index'
+import uuid from 'uuid/v4'
+
+const CalloutMenu = ({ menu, styles }) => {
+  const [showMenu, setMenu] = React.useState(false)
+  const toggleRef = React.useRef()
+
+  return menu ? (
+    <div id={uuid()} ref={toggleRef} {...styles}>
+      <MoreIcon
+        width='calc(1em + 7px)'
+        onClick={() => {
+          setMenu(!showMenu)
+        }}
+      />
+      <Callout toggleRef={toggleRef} expanded={showMenu} setExpanded={setMenu}>
+        {menu}
+      </Callout>
```

Comment:
Ideally we would only be rendering the callout when it's shown:
```suggestion
      {showMenu && <Callout>
        {menu}
      </Callout>}
```
(Instead of `display: expanded ? 'block' : 'none'` inside)

Reducing the amount of element on a long feed can have a substantial performance benefit.